libmount/src/context.c

```diff
@@ -320,6 +322,14 @@ int mnt_context_reset_status(struct libmnt_context *cxt)
 	return 0;
 }
 
+int mnt_context_get_linux_version(struct libmnt_context *cxt)
+{
+	if (cxt->lk_version == 0)
+		cxt->lk_version = get_linux_version();
```

Comment:
This is already cached inside `get_linux_version()` itself, so it shouldn't be necessary to cache it again.

Reply:
Oh, good catch.